username_api.py

```diff
@@ -43,6 +43,13 @@ def get_avatar(website, username):
         if not result or not result[0]:
             return None
         result = result[0]
+    elif data == 'openhub':
+        # Parse the Open Hub profile page
+        page = r.get(f'https://www.openhub.net/accounts/{username}')
```

Comment:
Get the website's URL from `websites.yml`, don't hard-code it.

Reply:
Fixed